docs/topics/minishift-install-nexus-setup.adoc

```diff
@@ -96,80 +99,40 @@ oc expose svc/nexus
-----
-+
-[NOTE]
---
-Nexus comes pre-configured for the Maven Central Repository, but you may need other repositories for your application. To access images provided by Red Hat, add the `jboss-ga` Maven repository to your Nexus instance.
---
-+
 .  Navigate to the project containing your Booster application.
 +
 [source,bash,subs="attributes+"]
 --
 oc project {project-name}
 --
 +
-. Assign the `admin` role to your Jenkins application and the `edit` permissions to your OpenShift account.
-Skip this step if you have already assigned these permissions.
+. Define and start a new build. Pass in a parameter to set the value of the `MAVEN_MIRROR_URL` to match the service URL of your Nexus application:
 +
-[source,bash,subs="attributes+"]
---
-oc policy add-role-to-user edit system:serviceaccount:{project-name}:default
-oc policy add-role-to-user admin system:serviceaccount:{project-name}:jenkins
---
-+
-. Obtain the Jenkins server URL and use it to access the Jenkins instance from your browser.
-+
-[source,bash,subs="attributes+"]
+[NOTE]
 --
-oc get routes jenkins
+Ensure that the YAML template of the builder image you are using for your application has the `MAVEN_MIRROR_URL` environmental variable defined.
+If it does not, see the Nexus documentation for instructions on link:https://help.sonatype.com/display/NXRM2/Maven+and+Other+Build+Tools[configuring your build manually^] before proceeding.
 --
 +
 [source,bash,subs="attributes+"]
 --
-NAME      HOST/PORT                                         PATH      SERVICES   PORT      TERMINATION
-jenkins   jenkins-{app-name}-{project-name}.{osl-route-hostname}             jenkins    <all>     edge/Redirect
+$ oc new-build {app-name}:latest~{scm-repo-url} \
+-e MAVEN_MIRROR_URL='http://nexus.{nexus-project-name}:8081/nexus/content/groups/public'
 --
 +
-. Navigate to `jenkins-{app-name}-{project-name}.{osl-route-hostname}/configure`, and log in using your OpenShift credentials to access the Jenkins configuration page.
-+
-. Go to the section titled _Kubernetes Pod Template_,  click on _Add Evironmental Variable_, and select _Global Environmental Variable_ option from the drop-down menu.
-+
-. Define a new environment variable:
-* Name: `MAVEN_MIRROR_URL`
-* Value: `nexus-{nexus-project-name}.{osl-route-hostname}/nexus/content/groups/public/`
-+
 [NOTE]
 --
-The actual value of the `MAVEN_MIRROR_URL` environmental variable corresponds to the service route of the pod running the Nexus server instance on your {OpenShiftLocal}.
-
-To obtain this route, navigate to the project that contains your Nexus instance and execute the command `oc get routes nexus`.
+Nexus comes pre-configured for the Maven Central Repository, but you may need other repositories for your application. To access images provided by Red Hat, add the link:https://maven.repository.redhat.com/ga/[Red Hat JBoss Middleware Maven Repository] to your Nexus instance.
```

Comment:
Actually I'm thinking the Beta artifacts are in m.r.r.c/earlyaccess, not m.r.r.c/ga. Maybe the link should point just to https://maven.repository.redhat.com/ which has a description.

Reply:
@Ladicek Good point. I'll go with the link to https://maven.repository.redhat.com/. Thanks:)